instrumentation/java-http-server/library/src/main/java/io/opentelemetry/instrumentation/javahttpserver/JavaHttpServerExchangeGetter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.javahttpserver;
+
+import com.sun.net.httpserver.HttpExchange;
+import io.opentelemetry.context.propagation.internal.ExtendedTextMapGetter;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import javax.annotation.Nullable;
+
+enum JavaHttpServerExchangeGetter implements ExtendedTextMapGetter<HttpExchange> {
+  INSTANCE;
+
+  @Override
+  public Iterable<String> keys(@Nullable HttpExchange exchange) {
+    if (exchange == null) {
+      return Collections.emptyList();
+    }
+    return exchange.getRequestHeaders().keySet();
+  }
+
+  @Nullable
+  @Override
+  public String get(@Nullable HttpExchange carrier, String key) {
+    if (carrier == null) {
+      return null;
+    }
+
+    List<String> list = carrier.getRequestHeaders().get(key);
+    return list != null ? list.get(0) : null;
+  }
+
+  @Override
+  public Iterator<String> getAll(@Nullable HttpExchange carrier, String key) {
+    if (carrier == null) {
+      return Collections.emptyIterator();
```

Comment:
`Collections.emptyIterator()` appears several times in the class, we'd use static import here, you can check other places and adjust them together. For detail, you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports